client_admin/pulp/client/admin/admin_auth.py

```diff
@@ -93,6 +121,22 @@ def login(self, **kwargs):
             msg = msg % {'e' : expiration_date}
 
         self.context.prompt.render_success_message(msg)
+    
+    def has_kerberos_ticket(self):
```

Comment:
This function appears to be identical to the has_kerberos_ticket() from pulp.bindings.server. Please reuse the function from there instead of repeating it.